ERROR - 2022-12-13 08:29:25 --> mysqli_sql_exception: Unknown column 'country' in 'field list' in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Country.php(72): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Country->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Country))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-13 08:30:04 --> mysqli_sql_exception: Unknown column 'country' in 'field list' in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Country.php(72): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Country->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Country))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-13 08:40:09 --> Too few arguments to function App\Controllers\Admin\Country::update(), 0 passed in C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php on line 904 and exactly 1 expected
in APPPATH\Controllers\Admin\Country.php on line 80.
 1 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Admin\Country->update()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Country))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-13 08:40:41 --> Too few arguments to function App\Controllers\Admin\Country::update(), 0 passed in C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php on line 904 and exactly 1 expected
in APPPATH\Controllers\Admin\Country.php on line 80.
 1 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Admin\Country->update()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Country))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
